docs/getting_started/hopsworksai/guides/eks_ecr_integration.rst

```diff
@@ -0,0 +1,327 @@
+    [ℹ]  if you encounter any issues, check CloudFormation console or try 'eksctl utils describe-stacks --region=us-east-2 --cluster=my-eks-cluster'
+    [ℹ]  CloudWatch logging will not be enabled for cluster "my-eks-cluster" in "us-east-2"
+    [ℹ]  you can enable it with 'eksctl utils update-cluster-logging --region=us-east-2 --cluster=my-eks-cluster'
+    [ℹ]  Kubernetes API endpoint access will use default of {publicAccess=true, privateAccess=false} for cluster "my-eks-cluster" in "us-east-2"
+    [ℹ]  2 sequential tasks: { create cluster control plane "my-eks-cluster", 2 sequential sub-tasks: { no tasks, create managed nodegroup "my-nodes" } }
+    [ℹ]  building cluster stack "eksctl-my-eks-cluster-cluster"
+    [ℹ]  deploying stack "eksctl-my-eks-cluster-cluster"
+    [ℹ]  building managed nodegroup stack "eksctl-my-eks-cluster-nodegroup-my-nodes"
+    [ℹ]  deploying stack "eksctl-my-eks-cluster-nodegroup-my-nodes"
+    [ℹ]  waiting for the control plane availability...
+    [✔]  saved kubeconfig as "/Users/maism/.kube/config"
+    [ℹ]  no tasks
+    [✔]  all EKS cluster resources for "my-eks-cluster" have been created
+    [ℹ]  nodegroup "my-nodes" has 2 node(s)
+    [ℹ]  node "ip-192-168-21-142.us-east-2.compute.internal" is ready
+    [ℹ]  node "ip-192-168-62-117.us-east-2.compute.internal" is ready
+    [ℹ]  waiting for at least 2 node(s) to become ready in "my-nodes"
+    [ℹ]  nodegroup "my-nodes" has 2 node(s)
+    [ℹ]  node "ip-192-168-21-142.us-east-2.compute.internal" is ready
+    [ℹ]  node "ip-192-168-62-117.us-east-2.compute.internal" is ready
+    [ℹ]  kubectl command should work with "/Users/maism/.kube/config", try 'kubectl get nodes'
+    [✔]  EKS cluster "my-eks-cluster" in "us-east-2" region is ready
+
+Once the cluster is created, eksctl will write the cluster credentials for the newly created cluster to your local kubeconfig file (~/.kube/config).
+To test the cluster credentials, you can run the following command to get list of nodes in the cluster. 
+
+.. code-block:: bash 
+    
+    kubectl get nodes 
+
+Output:
+
+.. code-block:: bash 
+
+    NAME                                           STATUS   ROLES    AGE     VERSION
+    ip-192-168-21-142.us-east-2.compute.internal   Ready    <none>   2m35s   v1.17.9-eks-4c6976
+    ip-192-168-62-117.us-east-2.compute.internal   Ready    <none>   2m34s   v1.17.9-eks-4c6976
+
+Step 2: Create an instance profile role on AWS
+----------------------------------------------
+You need to create an instance profile role to allow instances created by Hopsworks.ai to access EKS and ECR.
+Go to the Roles section of the IAM service in AWS management console by clicking `roles <https://console.aws.amazon.com/iam/home#/roles$new?step=type&roleType=aws&selectedService=EC2&selectedUseCase=EC2>`_. Click on *Next: Permissions*, *Next: Tags*, *Next: Review*, and then name your role and click *Create role*.
```

Comment:
I think there is a logical step here which is not very obvious for the non-tech people. You already mention the steps to click next on Permissions, Tags etc but you don't say that you need to Click on `Create Role`, Select `AWS Service` and `EC2`.

If I don't click on your link I don't know what to do.